Sources/IBLinterFrontend/Commands/ValidateCommand.swift

```diff
@@ -26,11 +26,14 @@ struct ValidateCommand: CommandProtocol {
             fatalError("\(workDirectoryString) is not directory.")
         }
 
-        let config = Config(options: options) ?? Config.default
+        var config = Config(options: options) ?? Config.default
```

Comment:
Maybe it is possible to do all the  job in constructor and keep let keyword

Reply:
I couldn't figure out how to make that work since the `included` property actually gets set in the `init(from decoder: Decoder)` method which is automatically called via the `YAMLDecoder` object. Let me know if you have any ideas and I can give it a try!